content/docs/pipelines/sdk/python-based-visualizations.md

```diff
@@ -58,9 +58,29 @@ arguments.
 6. Click **Generate Visualization**.
 7. View generated visualization by scrolling down.
 
+## Using TFX visualizations
+
+<video width="960" height="540" controls>
+  <source src="/docs/videos/tfdv_example_with_taxi_pipeline.webm" type="video/webm">
+</video>
+
+1. On Pipelines page, click **[Sample] Unified DSL - Taxi Tip Prediction Model Trainer** to open Pipeline Details page.
+2. On Pipeline Details page, click **Create run**.
+3. On Create run page,
+    * Use a run name and an experiment name of your choice.
+    * Note that in **pipeline-root** field, use a bucket you have permission to write to.
+    * Click **Start** to create the run.
+4. After the run is complete, on Run Details page, click any step, e.g., click the first step **csvexamplegen** as shown in the video above.
```

Comment:
s/,e.g.,/.For example,

It's best to use the full term, "for example", rather than the abbreviation. Many people are confused by Latin abbreviations, and many people don't know the difference between "i.e." and "e.g.".

Reply:
Done